src/lib/y2partitioner/actions/delete_bcache.rb

```diff
@@ -35,6 +35,28 @@ def initialize(*args)
 
     private
 
+      # @see DeleteDevice#errors
+      def errors
+        errors = super + [shared_cset]
+
+        errors.compact
+      end
+
+      # Error when the bcache shares cset with any other bcache.
+      #
+      # @see doc/bcache.md
+      # @return [String, nil] nil if there is no error
+      def shared_cset
+        return nil unless device.bcache_cset
+        return nil if single_bcache_cset? || !device.exists_in_probed?
+
+        # TRANSLATORS: Error when trying to existing bcache that shares caches.
```

Comment:
The comments targeting the translators are useful to tell them what the placeholders mean or whether something is a title or a pop-up. But at some point we started to introduce comments explaining what is already said in the string they have to translate. Pretty useless. :wink: